src/main/scala/jawnfs2/package.scala

```diff
@@ -87,27 +86,16 @@ package object jawnfs2 {
       * @tparam J the JSON AST to return
       * @return some parsed JSON value, or None if the source is empty
       */
-    def runJsonOption[J](implicit F: Sync[F], A: Absorbable[O], facade: RawFacade[J]): F[Option[J]] =
+    def runJsonOption[J](implicit F: Sync[F], A: Absorbable[O], facade: Facade[J]): F[Option[J]] =
       stream.parseJson(AsyncParser.SingleValue).compile.last
 
-    /**
-      * Parses the source to a single JSON value.
-      *
-      * @param facade the Jawn facade to materialize `J`
-      * @tparam J the JSON AST to return
-      * @return the parsed JSON value, or the facade's concept of jnull if the source is empty
-      */
-    @deprecated("Use runJsonOption.map(_.getOrElse(facade.jnull()))", "0.13.0")
-    def runJson[J](implicit F: Sync[F], A: Absorbable[O], facade: Facade[J]): F[J] =
```

Comment:
We could keep this alive by switching to `Facade.NoIndexFacade`, but it has been deprecated for a while.

Reply:
I don't think this is a use case that's ever really been properly supported by Jawn (and in my view that's the way it should be). As I understand it, `Facade` isn't intended to abstract over JSON representations generally, but to tell the parser how to construct an instance of a JSON representation given some text.

In some cases the JSON representation may include location information that would make it impossible to construct a JSON null value without a location. You could _capture_ that statically by only providing a `Facade` (previously `RawFacade`) instance for that kind of AST and a `Facade.NoIndexFacade` (previously just `Facade`) for everything else, but that feels kind of like a hack to me, and I think we should move in the direction of considering `NoIndexFacade` just an implementation helper, not something that should be used as a constraint or as the static type of an instance.

My vote would be to drop this method as soon as possible, and if necessary to use `Facade` and `facade.jnull(0)` (as if you're parsing a top-level JSON `null`) in the meantime.

Reply:
I agree.  This was an awkward lingering use of `Facade` when everything else moved to `RawFacade`.  I certainly wouldn't undeprecate it, and keeping the deprecation to help people migrate won't help since the types changed.  We're better off without it.